var/spack/repos/builtin/packages/aperture-photometry/package.py

```diff
@@ -13,7 +13,7 @@ class AperturePhotometry(Package):
     homepage = "http://www.aperturephotometry.org/aptool/"
     url      = "http://www.aperturephotometry.org/aptool/wp-content/plugins/download-monitor/download.php?id=1"
 
-    version('2.8.2', 'cb29eb39a630dc5d17c02fb824c69571fe1870a910a6acf9115c5f76fd89dd7e', extension='tar.gz')
+    version('2.8.2', '28ae136c708a3ebcb83632230e119a03ca1a65499006ab69dc76e21b4921f465', extension='tar.gz')
```

Comment:
It looks like this checksum is for 2.8.4, right? It's a shame we can't find a version-specific URL.
```suggestion
    version('2.8.4', '28ae136c708a3ebcb83632230e119a03ca1a65499006ab69dc76e21b4921f465', extension='tar.gz')
    version('2.8.2', 'cb29eb39a630dc5d17c02fb824c69571fe1870a910a6acf9115c5f76fd89dd7e', extension='tar.gz')
```

Reply:
```
[root@Estuary-CentOS8 ~]# spack checksum aperture-photometry 2.8.4 2.8.2


    version('2.8.4', sha256='28ae136c708a3ebcb83632230e119a03ca1a65499006ab69dc76e21b4921f465')
    version('2.8.2', sha256='28ae136c708a3ebcb83632230e119a03ca1a65499006ab69dc76e21b4921f465')
```

I'm little confusion of this result.
I think it may be some error by using `url      = "http://www.aperturephotometry.org/aptool/wp-content/plugins/download-monitor/download.php?id=1" `

Reply:
So I think there is some side effect by using `url      = "http://www.aperturephotometry.org/aptool/wp-content/plugins/download-monitor/download.php?id=1"`
And we shold maintain this package for it's each version?

Reply:
It looks like you can only download the latest version from the website. This is fine, as users may have tarballs of older versions cached in their downloads folder.